### ISSUE TITLE
Tabular format that matches cloc output

### ISSUE THREAD

**Issue by elindsey** [boyter/scc#103]
Today scc produces tabular output in this form:
```
$ scc
───────────────────────────────────────────────────────────────────────────────
Language                 Files     Lines     Code  Comments   Blanks Complexity
───────────────────────────────────────────────────────────────────────────────
Java                       482     59085    44910      4574     9601       3644
XML                         37      3648     3220       336       92          0
Properties File             13       215      160        32       23          0
```

whereas cloc uses a different column ordering:
```
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
C                               10           4680           6621          30812
```

Often `code` is the most significant column. I find having it on the far right much easier - it scans better, there's no hunting around within the table.

What are your thoughts on switching the default format, or barring that, having a cloc-tabular format that mimics cloc's ordering?

Specifically, I'm proposing a table format that goes: files, lines, blank, comment, code, complexity. It looks like most implementations other than tokei follow this format.

**Comment by boyter** [comment]
I don't have any issues with switching the output format. So long as the change is not applied to CSV formats there should be little issue as I hope nobody is using `cut` to pull out columns.

I think from memory when I started I just copied the tokei columns. However making it close to cloc seems like a reasonable thing to do as `scc` is starting to get some of its functionality.

If you want to do the change yourself let me know and make a PR. Otherwise ill probably look at it in a few days. Should be a simple one, but fiddly to ensure that the columns work correctly.
